docs/kernel-versions.md

```diff
@@ -64,6 +64,7 @@ BPF support for `sock_ops` | 4.13 | [`40304b2a1567`](https://git.kernel.org/cgit
 BPF support for skbs on sockets | 4.14 | [`b005fd189cec`](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=b005fd189cec9407b700599e1e80e0552446ee79)
 bpftool utility in kernel sources | 4.15 | [`71bb428fe2c1`](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=71bb428fe2c19512ac671d5ee16ef3e73e1b49a8)
 BPF attached to cgroups as device controller | 4.15 | [`ebc614f68736`](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=ebc614f687369f9df99828572b1d85a7c2de3d92)
+bpf2bpf function calls | 4.16 |  [`cc8b0b92a169`](https://git.kernel.org/cgit/linux/kernel/git/bpf/bpf-next.git/commit/?id=cc8b0b92a1699bc32f7fec71daa2bfc90de43a4d)
```

Comment:
Do you want to use the linus tree commit link?
https://github.com/torvalds/linux/commit/cc8b0b92a1699bc32f7fec71daa2bfc90de43a4d#diff-3f169a344ebaf82af97903746dc16e4d